lib/MusicBrainz/Server/Form/User/EditProfile.pm

```diff
@@ -2,14 +2,21 @@ package MusicBrainz::Server::Form::User::EditProfile;
 
 use HTML::FormHandler::Moose;
 use List::MoreUtils qw( any all );
-use MusicBrainz::Server::Form::Utils qw( language_options select_options_tree );
+use MusicBrainz::Server::Form::Utils qw( language_options select_options_tree validate_username);
 use MusicBrainz::Server::Translation qw( l ln );
 use MusicBrainz::Server::Validation qw( is_valid_url );
 
 extends 'MusicBrainz::Server::Form';
 
 has '+name' => ( default => 'profile' );
 
+has_field 'username' => (
+    type      => 'Text',
+    required  => 1,
```

Comment:
It looks like the `required` field actually breaks the profile page, because since there's no username field on the page, nothing is submitted and an invisible error occurs.